### ISSUE TITLE
multipath.conf: default sections misses bunch of keywords

### ISSUE THREAD

**Issue by omgold** [hercules-team/augeas#284]
In the lense for /etc/multipath.conf, in the defaults sections, the following keys are not handled, but according to the man page are understood by multipath (Centos6, package device-mapper-multipath-0.4.9-87.el6.x86_64):

```
   multipath_dir
   find_multipaths
   queue_without_daemon
   flush_on_last_del
   checker_timeout
   log_checker_err
   hwtable_regex_match
   reservation_key
   retain_attached_hw_handler
   detect_prio
   reload_readwrite
   replace_wwid_whitespace
   force_sync
   config_dir
   delay_watch_checks
   delay_wait_checks
```

**Comment by raphink** [comment]
The lens for multipath is typed. I would encourage you to open a Pull Request to add support for these keywords with the proper typing, as well as corresponding tests.

**Comment by XMol** [comment]
Submitted PR #441 that was made with the man page of multipath 0.4.9-99.el7 as reference.

**Comment by XMol** [comment]
I don't know what to do to make the Travis build test succeed. As far as I understand, a test with lens-multipath.sh has failed, but where does this come from?

**Comment by domcleal** [comment]
Fixed in #441, thanks @XMol.
